rancher/dashboard
# Prevent blocking effect secondary resources on create/edit views

Fixes #6678 (`workload`, `persistentvolume` and `persistentvolumeclaim` create/edit views optimisation)

- Adds new `resource-manager` mixin, which should be used to handle requests for secondary resources on views
- Optimises fetching of secondary resources for `workload`, `persistentvolume` and `persistentvolumeclaim` create/edit views 

Testing scenarios:
- Make sure create/edit views works as expected 
- Check that secondary data is being fetched
- Make sure that secondary data is namespaced (only for `workload`)
- When changing `namespace` on `create` view, make sure there are new requests for secondary data with the correct namespace (only for `workload`)
